src/sql/util.py

```diff
@@ -404,3 +405,19 @@ def if_substring_exists(string, substrings):
     """Function to check if any of substring in
     substrings exist in string"""
     return any(msg in string for msg in substrings)
+
+
+def enclose_table_with_double_quotations(table):
```

Comment:
not all databases use double quotes to identify table names, we already have some utility to detect this: https://github.com/search?q=repo%3Aploomber%2Fjupysql%20is_use_backtick_template&type=code

Reply:
please use the is_use_backtick_template function to determine which character to use

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/855/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R423-R425)